health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/accessbilityPopUP.js

```
@@ -0,0 +1,104 @@
+  };
+
+  const optionsList3 = [
+    { name: "OTHER", code: "OTHER" },
+
+  ];
+
+  return (
+    <PopUp
+      onClose={onClose}
+      heading={t(`HCM_MICROPLAN_VILLAGE_ACCESSIBILITY_LABEL`)}
+      children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)